src/apiCalls.js

```diff
@@ -1,7 +1,12 @@
 export const getAllMovies = () => {
   return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
-    .then(response => response.json())
-    .then(data => data.movies);
+  .then(response => {
+    if (response.ok) {
+      return response.json();
+    } else {
+      throw new Error(response.statusText);
+    }
+  })
```

Comment:
Good checking for status codes to throw an error!  Once you start doing Cypress testing this would be a great thing to test and confirm that its doing what you expect (your stubbed response would need to include the `response.ok` property)